src/port1.0/portconfigure.tcl

```diff
@@ -857,12 +857,12 @@ proc portconfigure::max_version {verA verB} {
 # https://trac.macports.org/wiki/XcodeVersionInfo
 #----------------------------------------------------------------
 #| OpenMP Version |  Clang  |  Xcode Clang  |  Xcode  |   GCC   |
-#|---------------------------------------------------------------
-#|      2.5       |   3.8   |    Future?    | Future? |   4.2   |
-#|      3.0       |   3.8   |    Future?    | Future? |   4.4   |
-#|      3.1       |   3.8   |    Future?    | Future? |   4.7   |
-#|      4.0       | Partial |    Future?    | Future? |   4.9   |
-#|      4.5       | Partial |    Future?    | Future? |   ???   |
+#|--------------------------------------------------------------|
+#|      2.5       |   3.8   |   703.0.29    |   7.3   |   4.2   |
+#|      3.0       |   3.8   |   703.0.29    |   7.3   |   4.4   |
+#|      3.1       |   3.8   |   703.0.29    |   7.3   |   4.7   |
+#|      4.0       |   6.0   | 1000.11.45.2  |  10.0   |   4.9   |
+#|      4.5       |   6.0   | 1000.11.45.2  |  10.0   |    8    |
 #----------------------------------------------------------------
```

Comment:
The Xcode clang versions are inferred from the corresponding upstream LLVM versions as listed at https://en.wikipedia.org/wiki/Xcode#Toolchain_versions. I may want to ask @mascguy or someone else with access to older macOS/Xcode to confirm these cutoffs (which should only require compiling a sample OpenMP program).

Also, I notice that Clang 6 is allowed for OpenMP > 3.1; was doing so provisional? Only Clang 7 and later claim to fully support OpenMP 4.5, so would it be better to raise the requirement to Clang 7 (Xcode 10.2)?

I presume this table should be kept synchronized with https://trac.macports.org/wiki/CompilerSelection#Parallelism